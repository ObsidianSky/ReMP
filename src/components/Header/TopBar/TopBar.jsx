import React from 'react';
import { Route } from 'react-router-dom';

if (process.env.BROWSER) {
	require('./TopBar.scss');
}
import Logo from '../../common/Logo/Logo';
import HomeButton from './HomeButton/HomeButton';

const TopBar = () => (
    <div className="top-bar">
        <Logo/>
        <Route path="/film" component={HomeButton}/>
    </div>
);

export default TopBar;