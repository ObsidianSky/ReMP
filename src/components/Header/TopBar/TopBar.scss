@import '../../../assets/styles/variables';

.top-bar {
  padding: 5px 0;
  color: $pink;
  display: flex;
  justify-content: space-between;
  align-items: center;

  &__brand {
    font-size: 25px;
  }
}