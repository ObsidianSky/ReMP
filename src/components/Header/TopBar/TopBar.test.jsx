import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TopBar from './TopBar';

describe('Header', () => {

    test('should renders correctly', () => {
        const wrapper = shallow(<TopBar />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

