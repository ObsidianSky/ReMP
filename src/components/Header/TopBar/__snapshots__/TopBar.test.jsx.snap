// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders correctly 1`] = `
<div
  className="top-bar"
>
  <Logo />
  <Route
    component={[Function]}
    path="/film"
  />
</div>
`;
